simln-lib/Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde_millis = "0.1.1"
 rand_distr = "0.4.3"
 mockall = "0.12.1"
 rand_chacha = "0.3.1"
+tokio-util = { version = "0.7.13", features = ["rt"] }
```

Comment:
nittynitnit: try to limit the [commit message's title to 72 chars](https://www.gitkraken.com/learn/git/best-practices/git-commit-message#git-commit-message-structure)